task/09-functions-n-closures-tasks.js

```diff
@@ -131,7 +131,17 @@ function memoize(func) {
  * retryer() => 2
  */
 function retry(func, attempts) {
-    throw new Error('Not implemented');
+    let attempt_number = 1,
+        error;
+
+    return function () {
+        while (attempt_number <= attempts) {
+            try {return func()}
+            catch (Error) {error = Error} 
+            attempt_number++;
+        }
+        throw error
```

Comment:
error variable is redundant
You can pass argument to catch statment and throw it then:
`catch(e){
throw e;
}`